examples/demos/elation/Elation.jsx

```diff
@@ -90,15 +90,15 @@ export default class Elation extends React.Component {
         titleAccessor="_patientName"
         startAccessor={(event) => new Date(event._apptTime)}
         endAccessor={(event) => new Date(event._apptEnd)}
-        availabilityStartAccessor={({availability, date}) => {
-          const availabilityDayStart = new Date(date);
-          availabilityDayStart.setHours(...availability.startTime.split(':'));
-          return availabilityDayStart;
+        availabilityStartAccessor={(availability) => {
+          const startDate = new Date(2015, 3, 12);
```

Comment:
No-op:

Just a note if you weren't already on this train of thought - I think the prop design here means the EMR app will have to generate individual instances of availabilities per availability object. For example if an availability is "Wednesdays from 9am-7pm", that doesn't inherently correspond to a specific date. The EMR will have to generate individual instances, e.g. [first Wednesday of the month from 9am-7pm, second Wednesday of the month from 9am-7pm, third Wednesday of the month from 9am-7pm]. This would be similar to how the EMR generates instances of recurring events.

I think this is _correct/ideal_ behavior btw, not suggesting we should change this. It makes availabilities conceptually consistent with events, and doesn't paint us into a corner if we ever want to support one-off availabilities.

Reply:
Yup, that makes sense!